src/table/components/head/__tests__/TableHeadWrapper.spec.tsx

```diff
@@ -82,7 +82,7 @@ describe('<TableHeadWrapper />', () => {
     expect(queryByText(tableData.columns[1].label)).toBeVisible();
   });
 
-  it.skip('should show the menu button when the head cell is on focus', () => {
+  it('should show the menu button when the head cell is on focus', () => {
```

Comment:
I did whatever I could to make it work but it doesn't : ( 
Hovering and focusing works properly on the browser but can't verify it here that it make the button visible through this test. I tried with userEvent.hover or tab and got the same result. It doesn't show that the opacity is changing to '1'.

Reply:
This might be a timing issue. Does the first `expect` pass? If so, try a small manual timeout before the second `expect`. If that works we need to figure out a proper way to do the timeout

Reply:
If it's a timing issue you could try to wrap you `expect` in a `waitFor()` callback

Reply:
nice!